### DISCUSSION TITLE
Errore 422 creazione fattura

### DISCUSSION THREAD

**Discussion by Formedialecce** [fattureincloud/api#280] [Q&A] [answered]
Sto provando a crare una fattura utilizzando le api 2. Ma mentre riesco a far funzionare tutto da postman da codice php invece ricevo un errore 422 ... come se i dati postati non fossero formattati correttamente 

di seguito vi scrivo il codice che uso 

` $curl_handle = curl_init();
                        curl_setopt($curl_handle, CURLOPT_URL, 'https://api-v2.fattureincloud.it/c/MIO COMPANY ID/issued_documents');
                        curl_setopt($curl_handle, CURLOPT_RETURNTRANSFER, true);
                        curl_setopt($curl_handle, CURLOPT_ENCODING, '');
                        curl_setopt($curl_handle, CURLOPT_POST, 1);
                        curl_setopt($curl_handle, CURLOPT_HEADER, true);
                        curl_setopt($curl_handle, CURLOPT_HTTPHEADER, array(
                            'Accept:application/json',
                            'Authorization:Bearer ' . FIC_TOKEN
                        ));
                        curl_setopt($curl_handle, CURLOPT_POSTFIELDS,$data_string);

                        $response = curl_exec($curl_handle);
                        //curl_close($curl_handle);

                        $result = json_decode($response);`

ll json $data_string è corretto perchè se lo uso da postman la fattura viene creata
Dove sto sbagliando ?

**Comment by valmoz** [comment, marked as answer]
Ciao @Formedialecce 
posso chiederle perché non sta usando la nostra [PHP SDK](https://developers.fattureincloud.it/docs/sdks/php-sdk)?

Grazie
Mauro
